content/02.introduction.md

```diff
@@ -2,5 +2,94 @@
 Set introduction for paper here
 Talk about problem, goal, and significance of paper
 
-## Recent Work
-Talk about what has been done in the field in regards to text mining and knowledge base integration
+### Related Work
+
+Relationship extraction is the process of detecting and classifying semantic relationships from a collection of text.
+This process can be broken down into three different categories: (1) the use of natural language processing techniques such as manually crafted rules and identifying key text patterns for relationship extraction, (2) the use of unsupervised methods via co-occurrence scores or clustering, and (3) supervised or semi-supervised machine learning using annotated datasets for classification of documents or sentences.
+In this section, we discuss selected efforts for each type of edge that we included in this project.
```

Comment:
```suggestion
In this section, we discuss selected efforts for each type of edge that we include in this project.
```
I realize that I changed this to past tense before, but I'm just realizing that it's the intro and should be present tense. Sorry! 😆